datafusion/physical-plan/src/execution_plan.rs

```diff
@@ -881,6 +907,16 @@ impl PlanProperties {
         self
     }
 
+    pub fn with_scheduling_type(mut self, scheduling_type: SchedulingType) -> Self {
```

Comment:
I think it would help to document the defaults here

```suggestion
    /// Set the [`SchedulingType`]. 
    ///
    /// Defaults to [`SchedulingType::Blocking`]
    pub fn with_scheduling_type(mut self, scheduling_type: SchedulingType) -> Self {
```